compiler/compile/OMRCompilation.hpp

```diff
@@ -969,15 +965,15 @@ class OMR_EXTENSIBLE Compilation
    void reportFailure(const char *reason);
 
 protected:
+   // Signature of the method we are compiling -- keep this at offset 0
```

Comment:
In the interests of moving this forward and merging, please remove the comment unless there is a justifiable, hard requirement for this field to be the first one in this class in OMR.  I don't believe there is.